pypeit/core/meta.py

```diff
@@ -164,6 +164,7 @@ def define_additional_meta(nlamps=20):
                        'slitlength': dict(dtype=float, comment='Slit length, used only for long slits'),
                        'temperature': dict(dtype=float, comment='Temperature (units.K) at observation time'),
                        'utc': dict(dtype=str, comment='UTC of observation'),
+                       'mirror': dict(dtype=str, comment='Position of the mirror (e.g. IN or OUT)'),
```

Comment:
What mirror?

Reply:
good question!  I don't know..  

I've edited it to be more generic.

Reply:
Just saw this stop by the e-mail and I actually have an answer. The mirror is used to toggle between viewing internal calibrations vs viewing sky. When the mirror is "in" the light source are the calibration lamps, which is usually when taking arcs or internal flats (the code uses it to distinguish twilight flats from internal flats). When the mirror is "out", the light source is the sky, so primarily for science images. I hope that makes sense!